### DISCUSSION TITLE
vSphere UPI install VIP api load balancer ip crash

### DISCUSSION THREAD

**Discussion by Hyun-June-Choi** [okd-project/okd#1277] [Q&A]
.100 : bootstrap
.92 : api load balancer

after bootstrap ignition complete, API load balancer IP is set to bridge network as shown below
Subsequently, traffic from the API load balancer to the bootstrap is blocked(always).

<img width="515" alt="image" src="https://user-images.githubusercontent.com/16183877/177856399-b9db5bc1-4dc6-484f-8bda-82293a6ab310.png">

What's wrong?

**Comment by jcpowermac** [comment]
OKD Version?

If in your install-config you are setting the vips you will get the static pods that make up the networking features we need for IPI.
So that includes keepalived.

You might be missing a extra-config. Take a look:
https://github.com/jcpowermac/openshift-upi-powercli/blob/main/upi.ps1#L147-L148

And from my perspective your screenshot looks correct. I expect keepalived to start up on the boostrap node and add `.92`
It should move to one of the masters after bootstrap is complete.

**Reply by Hyun-June-Choi** [reply]
4.10.0-0.okd-2022-06-10-131327
environment without dhcp.
At least it seems to me that the other version is the same.

**Reply by jcpowermac** [reply]
I can try tomorrow but the way you are installing will require dhcp.

**Reply by Hyun-June-Choi** [reply]
Did I misunderstand the manual?
Is dhcp a must?

**Reply by Hyun-June-Choi** [reply]
I added dhcp but there is no change

There are similar issues, but they seem to be resolved in the previous version.
https://github.com/openshift/machine-config-operator/pull/2586

**Reply by Hyun-June-Choi** [reply]
I have an idea after looking at your opinion and the link below. I want to put down the load balancer when I install it.

https://learn.redhat.com/t5/Containers-DevOps-OpenShift/Openshift-IPI-on-VMware/td-p/15358
